prow/cluster/cherrypicker_deployment.yaml

```diff
@@ -24,6 +24,7 @@ spec:
         - --dry-run=false
         - --use-prow-assignments=false
         - "--labels=cla: yes"
+        - --labels=release-notes-none
```

Comment:
Any idea why the prior one is in quotes and this one is not?

Reply:
I guess the space would change it.

Reply:
space. it could be the binary would interpret `yes` as another argument.